src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,128 @@
+    }
+
+    @Test
+    void register_wrongPasswordsLength_NotOk() {
+        actual = new User(DEFAULT_LOGIN, WRONG_PASSWORD, DEFAULT_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_wrongAge_NotOk() {
+        actual = new User(DEFAULT_LOGIN, DEFAULT_PASSWORD, WRONG_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_loginExists_NotOk() {
+        actual = new User(LOGIN_EXISTED_IN_STORAGE, DEFAULT_PASSWORD, DEFAULT_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_loginAndPasswordWrongValues_NotOk() {
+        actual = new User(null, WRONG_PASSWORD, DEFAULT_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_loginAndAgeWrongValues_NotOk() {
+        actual = new User(null, DEFAULT_PASSWORD, WRONG_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_loginExistsAndPasswordShort_NotOk() {
+        actual = new User(LOGIN_EXISTED_IN_STORAGE, WRONG_PASSWORD, DEFAULT_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_loginExistsAndAgeWrongValue_NotOk() {
+        actual = new User(LOGIN_EXISTED_IN_STORAGE, DEFAULT_PASSWORD, WRONG_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_passwordAndAgeWrongValue_NotOk() {
+        actual = new User(DEFAULT_LOGIN, WRONG_PASSWORD, WRONG_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_loginNullAndOtherWrongValues_NotOk() {
+        actual = new User(null, WRONG_PASSWORD, WRONG_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_loginExistsAndOtherWrongValues_NotOk() {
+        actual = new User(LOGIN_EXISTED_IN_STORAGE, WRONG_PASSWORD, WRONG_AGE);
+        assertThrows(InvalidInputDataException.class, () ->
+                registrationService.register(actual), EXCEPTION);
+    }
+
+    @Test
+    void register_validLogin_Ok() {
+        actual = new User("DEFAULT_LOGIN", "DEFAULT_PASSWORD", DEFAULT_AGE);
+        Assertions.assertEquals(actual, registrationService.register(actual),
+                "User added");
+    }

```

Comment:
method work incorrect
you should create new user then you call method register on this user 
and then you should check if user was added to storage using method get this user will be actual
and finaly in assertEquals you should compare user and actual